src/routes/SwitchDepartment/container.js

```diff
@@ -28,7 +28,7 @@ export class SwitchDepartmentContainer extends Component {
 
 		if (!room) {
 			const user = await Livechat.grantVisitor({ visitor: { department, token } });
-			await dispatch({ user, alerts: (alerts.push({ id: createToken(), children: I18n.t('Department switched'), success: true }), alerts) });
+			await dispatch({ department, user, alerts: (alerts.push({ id: createToken(), children: I18n.t('Department switched'), success: true }), alerts) });
```

Comment:
This change won't impact anywhere.
As you can see [here](https://github.com/RocketChat/Rocket.Chat.Livechat/blob/develop/src/lib/hooks.js#L68), when you register the department, we update the `iframe` prop, not the root of the object.